build.gradle

```diff
@@ -24,6 +24,9 @@ allprojects {
             url "https://dl.bintray.com/algolia/maven"
         }
     }
+    buildscript {
+        apply plugin: 'com.github.ben-manes.versions'
```

Comment:
What's this for

Reply:
This allow a new command : `./gradlew dependencyUpdates` which will check all new available dependencies. Very useful to know what could be updated.